index.html

```diff
@@ -34,25 +32,32 @@ <h1>Test Patterns for Review</h1>
         <th>Pattern</th>
         <th>Index Page</th>
         <th>Review Page</th>
-        <th>Number of Tests</th>
+        <th># of Tests</th>
+        <th>Commit of Last Change</th>
       </tr>
           <tr>
             <td>checkbox</td>
-            <td><a href="./tests/checkbox/index.html">Index</a></td>
-            <td><a href="./review/checkbox.html">Review</a></td>
+            <td><a class="githack-link" href="./tests/checkbox/index.html">Index</a></td>
+            <td><a class="githack-link" href="./review/checkbox.html">Review</a></td>
```

Comment:
What is this class for?

Reply:
I added it in for an attempted fix where I added a script that targeted this class. Removed it now.